miranda/structure/_structure.py

```diff
@@ -78,89 +78,75 @@ def build_path_from_schema(
     -------
     Path or None
     """
+    folder_tree_structure = None
     try:
         if facets["type"] == "station-obs":
             STATION_OBS_SCHEMA.validate(facets)
-            folder_tree = (
-                Path(output_folder)
-                / facets["type"]
-                / facets["project"]
-                / facets["institution"]
-                / facets["version"]  # This suggests "date_created"
-                / facets["frequency"]
-                / facets["variable"]
+            folder_tree_structure = (
+                "type",
+                "institution",
+                "version",  # This suggests "date_created"
+                "frequency",
+                "member"
+                if hasattr(facets, "member")
+                else None,  # This suggests station code
```

Comment:
ah cool! If there is no member, the level is removed in the tree ?

Reply:
Precisely!